docs/plugins/playlist.rst

```diff
@@ -24,6 +24,16 @@ name::
 
     $ beet ls playlist:anotherplaylist
 
+A playlist query will use the paths found in the playlist file to match items
+in the beets library. Since `playlist:` submits a regular beets query like any
+other query syntax (eg. `artist:`, `year:`, ...), the sorting order of either
+the configuartion setting `sort_item` or any sorting patterns submitted with
+the query command will be used::
```

Comment:
Maybe this should be phrased as: "If you want the list in any particular order, you can use the standard beets query syntax for sorting:"